CHANGELOG.mdx

```diff
@@ -12,10 +12,17 @@ file is managed by ``towncrier``.
 
 You **may** edit previous change logs to fix problems like typo corrections or such.
 You can find more information on how to add a new change log entry at
-https://github.com/RasaHQ/rasa/tree/master/changelog/ . -->
+https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+
+## [2.3.3-bf.1] - 2021-02-25
+
+### Improvements
+
+- [#39](https://github.com/botfront/rasa-for-botfront/pull/40): update rasa to 2.3.3
```

Comment:
[#43](https://github.com/botfront/rasa-for-botfront/pull/43): Support for securing Rasa API with auth token with an env var.